rakefile.rb

```diff
@@ -168,9 +168,9 @@ def sanitized_version(version)
 def update_package_version(version) 
   #Replace token Version: x.y.z with the version from appveyor
   replacement = {
-    /Version: \d\.\d\.\d/ => "Version: #{version}"
+    /Version: \d+\.\d+\.\d+/ => "Version: #{version}"

```

Comment:
Reged. .was parsing 9.x.z but not 10.x.z...

Reply:
lol